### DISCUSSION TITLE
Possible doc correction for additional_hostnames

### DISCUSSION THREAD

**Discussion by skeemer** [ddev/ddev#4480] [General]
https://github.com/drud/ddev/blob/8fa51f444a88ee95fa2535b563a234b377c12a4c/docs/content/users/extend/additional-hostnames.md?plain=1#L13

I'm not sure if this should be escaped. It renders with the slash visible and I had a hard time getting it working because I kept trying to include it.

**Comment by rfay** [comment]
That is in fact incorrect, and a PR would be appreciated (just click the pencil in the upper right of the docs page). 

But preview it before you push the button to submit. I imagine it got that way because markdown was eating the `*`
